extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -79,7 +79,7 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 	// 3. ask for object name for the binding
 	const objectName = await vscode.window.showInputBox({
 		prompt: selectedBinding.type === BindingType.input ? constants.sqlTableOrViewToQuery : constants.sqlTableToUpsert,
-		value: constants.placeHolderObject,
+		placeHolder: constants.placeHolderObject,
```

Comment:
Would be good to validate it's non-empty as well.

Reply:
added validation 
![image](https://user-images.githubusercontent.com/31145923/135937917-97aeec7d-14fb-4623-9d23-beb56170b38f.png)